lib/swoosh/adapters/smtp/helpers.ex

```diff
@@ -66,6 +66,17 @@ if Code.ensure_loaded?(:mimemail) do
       Map.to_list(additional_headers) ++ headers
     end
 
+    defp prepare_parts(headers, %Email{
```

Comment:
It would be nice if we supported boundaries as described in https://tools.ietf.org/html/rfc1341